pkg/plugins/policies/meshtimeout/plugin/v1alpha1/plugin.go

```diff
@@ -1,17 +1,29 @@
 package v1alpha1
 
 import (
-	"github.com/kumahq/kuma/pkg/core"
+	"context"
+
+	envoy_cluster "github.com/envoyproxy/go-control-plane/envoy/config/cluster/v3"
+	envoy_listener "github.com/envoyproxy/go-control-plane/envoy/config/listener/v3"
+	envoy_route "github.com/envoyproxy/go-control-plane/envoy/config/route/v3"
+
+	mesh_proto "github.com/kumahq/kuma/api/mesh/v1alpha1"
 	core_plugins "github.com/kumahq/kuma/pkg/core/plugins"
 	core_mesh "github.com/kumahq/kuma/pkg/core/resources/apis/mesh"
+	"github.com/kumahq/kuma/pkg/core/xds"
 	core_xds "github.com/kumahq/kuma/pkg/core/xds"
```

Comment:
Two imports for the same modules (one named, one not)

Reply:
done